luatest/utils.lua

```diff
@@ -148,4 +149,10 @@ function utils.get_fn_location(fn)
     return ('%s:%s'):format(fn_source[#fn_source], fn_details.linedefined)
 end
 
+function utils.generate_id(length, urlsafe)
+    urlsafe = urlsafe or true
+    length = length or 9
```

Comment:
Honestly, for me, it looks so-so. Let's use the following approach:

```lua
if not length then length = 9 end
```

Reply:
Yep, fixed `length` and `urlsafe` params